Python.md

```diff
@@ -15,23 +20,63 @@ As most of the other projects written in Python, we use [PEP 8]
 (https://www.youtube.com/watch?v=wf-BqAjZb8M)" by Raymond Hettinger at PyCon 2015,
 which talks about the famous P versus NP problem.*
 
+If the project has a `pylintrc` file, you can use pylint to check for potential style violations:
+
+    pylint --rcfile=pylintrc
+
 The general idea is to make the code within a project consistent and easy to interpret (for humans).
```

Comment:
I wonder if we should add a section here with "common things to keep in mind":

* in dicts, no space before `:`, but space after
* in function definitions/calls, no space around `=`
* space after `,`
* no trailing whitespace
* no spaces before `(` function call or `[` dict/list access
* trailing commas
I know that these are things which are in pep-8, but these are the things that I spend a huge amount of time reviewing on PRs. I'd much rather be able to link to this section and say "fix your formatting"